Added Background introduction  and modified introduction to analogue sim...

MikeBrinson opened the pull request:
...ulation.

MikeBrinson committed:
Added Background introduction  and modified introduction to analogue simulation.

guitorri commented on source/index.rst:
```diff
@@ -7,11 +7,17 @@
 Welcome to Qucs Help's documentation!
 =====================================
 
+.. row of flags, alphabetical 
```
This is now removed, flags was a bad idea. One languge can be used in many places

guitorri commented on source/index.rst:
```diff
@@ -7,11 +7,17 @@
 
+.. row of flags, alphabetical 
+
+|flab_cs| |flab_de| |flag_en| |flab_es| |flab_fr| |flab_pt| |flab_ru| |flab_uk|
```
No more flags

guitorri commented on source/index.rst:
```diff
@@ -34,11 +40,36 @@ Example schematics
     Available online at http://qucs.sourceforge.net/download.html#example
 
 
```
Add 
.. only :: html

guitorri commented on source/index.rst:
```diff
@@ -34,11 +40,36 @@ Example schematics
 
-  Indices and tables
-  ==================
+.. |flab_pt| image:: _static/flagicons/br.png
```
pt is now pt_BR

guitorri commented on source/start.rst:
```diff
@@ -101,7 +36,7 @@ Wonderful, give yourself a clap on the shoulder!
 .. |image0_FR| image:: _static/fr/qucsmain.png
 
-.. |image0_PT| image:: _static/pt_BR/qucsmain.png
+.. |image0_PT| image:: _static/pt/qucsmain.png
```
pt is now pt_BR

guitorri commented:
Hi Mike.  Please delete the `DEADJOE` file and update your branch. Just do your changes and push again to the same branch on your fork. The pull request will be updated.

MikeBrinson committed:
More changes to background and intro to analogue simulation.

MikeBrinson committed:
More intro and start changes

guitorri committed:
index: remove flags, indices only on html

* flags was a bad idea, readthedocs use the language code which is
  better.

guitorri committed:
start: fix figure references

* During translation the tags are translated
    |image1_EN| is translated to |image1_FR|
  Doing so Sphinx is capable os pickin up the righ images

* Eventually we will have to put figures into directories.
  For now using the same prefix as the source .rst helps a bit.

guitorri committed:
start: use verbatim for commands and system directories

guitorri committed:
intro: fix link

guitorri committed:
intro: balance size of paragraphs

guitorri committed:
intro: there is no isonum.txt to include

guitorri commented:
@MikeBrinson  I guess we can close this pull request, right?

MikeBrinson committed:
Merge pull request #2 from guitorri/guitorri-rev

First revision of new documentation

MikeBrinson commented:
Hello Guilherme
Yes - please close pull request.
More changes and additions to follow in thenext stage of document  preparation.
Best wishes.
Mike
 Mike Brinson

mbrin72043@yahoo.co.uk
      From: Guilherme Brondani Torri notifications@github.com
 To: Qucs/qucs-help qucs-help@noreply.github.com 
Cc: MikeBrinson mbrin72043@yahoo.co.uk 
 Sent: Wednesday, 21 January 2015, 22:53
 Subject: Re: [qucs-help] Added Background introduction and modified introduction to analogue sim... (#1)

@MikeBrinson I guess we can close this pull request, right?—
Reply to this email directly or view it on GitHub.
